test/Microsoft.Sbom.Targets.E2E.Tests/GenerateSbomE2ETests.cs

```diff
@@ -19,6 +18,10 @@ public class GenerateSbomE2ETests
     /*
      * The following tests validate the end-to-end workflow for importing the Microsoft.Sbom.Targets.targets
      * into a .NET project, building it, packing it, and validating the generated SBOM contents.
+     *
+     * NOTE: These tests are only compatible with net6.0 and net472, as there are issues when resolving NuGet assemblies when
+     * targeting net8.0. There seems to have been a fix implemented but has not been released for the MSBuild locator package:
+     * https://github.com/microsoft/MSBuildLocator/issues/86
```

Comment:
- [ ] Update this comment to not mention the MSBuildLocator issue, since it looks like there is already a fix for our issue.